### PR TITLE
docs: init Rslib website

### PR DESCRIPTION
**Pull request by Timeless0911**
## Summary

Rslib website init

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "version": "0.0.1",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "build": "rspress build",
+    "dev": "rspress dev",
+    "preview": "rspress preview"
+  },
+  "devDependencies": {
+    "@rsbuild/core": "1.0.1-rc.5",
```
Comment (fi3ework) [inline website/package.json:12]: @rsbuild/core should not be here?

Comment (Timeless0911) [reply website/package.json:12]: use `/// <reference types="@rsbuild/core/types" />` in `env.d.ts` to declare assets module

**Comment by fi3ework** [comment]
Basically LGTM, should we first deploy a temporary version on Netlify to facilitate the use of Netlify's preview link, and then deploy it to rslib.rsbuild.dev when it is stable?

**Comment by fi3ework** [comment]
We can test preview build in following PRs after the build workflow in this PR merged.

**Review by fi3ework — APPROVED** [review]

**Pull request merged**